### DISCUSSION TITLE
Do we still have Godot 2.x PRs coming?

### DISCUSSION THREAD

**Discussion by SysError99** [godotengine/godot-proposals#7369] [General Discussions]
I'm curious if there are any of Godot 2.x going around and especially being accepted into the branch (just without porting). I'm about to suggest a full proposal if it will apply to upcoming 3.x for even more further 3.x lifespan and easier maintenance.

**Comment by AThousandShips** [comment]
2.x is no longer supported, and haven't been for ~4 years (2019, my bad confused the first release of 2.1 with the most recent), though the last commits on that branch was back in 2021 it was just the last gasp

**Reply by AThousandShips** [reply]
On the topic of 3.x however 3.x *specific* features or improvements, that are *not* bugfixes or security updates, are very unlikely to get approved, as it is in LTS and will only be maintained as opposed to developed, again with the exception of features that can be backported

See [here](https://docs.godotengine.org/en/stable/about/release_policy.html#release-support-timeline)

Now as I understand it 3.x will be supported and maintained indefinitely, on a best effort basis, meaning:
* As long as there are people needing it
* As long as there are people able to do it

I would suspect that in the next year/year and a half most new projects using Godot will have transitioned to 4.x, as the last features currently missing or limited as compared with 3.x are added, and there will be no real reason to develop new projects on 3.x (save for allowing low end support, which remains a valid reason for using 3.x)

This will mean that most projects using it out there will be released and maintained projects, mainly needing bug fixes and security updates, as opposed to new features (as the games will already be shipped, and updates to them should be minor)

With less use of it, especially fewer people developing new games with it, the number of bugs that will be found will gradually drop

**Comment by tetrapod00** [comment]
We actually did get one more 2.x PR recently, just to keep it compiling on modern toolchains: https://github.com/godotengine/godot/pull/101040.

But since in general the answer is "no", besides that single novelty, I'll close this as resolved.
